extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -538,6 +541,38 @@ public String getUploaderUrl() throws ParsingException {
         throw new ParsingException("Could not get uploader url");
     }
 
+    @Override
+    public String getTopicUrl() throws ParsingException {
+        // This is not in the api yet topicDetails is empty
+        // https://developers.google.com/youtube/v3/docs/videos/list
+        String url = getUrl();
+        try {
+            String data = "";
+            Document doc = Jsoup.connect(url).get();
+            org.jsoup.select.Elements el = doc.select("script");
```

Comment:
You should not make another request. Surely the topic url information is stored in one of the `JsonObject` fields of this class (probably `nextResponse`). You can probably replace `json` below with `nextResponse`.